### PR TITLE
SDK-648 fix tests post gradle

### PR DESCRIPTION
**Pull request by kfrollo**

### CODE REVIEW

File: src/androidTest/kotlin/us/frollo/frollosdk/user/UserManagementTest.kt [hunk 1]
```diff
@@ -1225,6 +1226,7 @@ class UserManagementTest : BaseAndroidTest() {
         tearDown()
     }
 
+    @Ignore("temp")
```
Comment (deepakmn86) [inline src/androidTest/kotlin/us/frollo/frollosdk/user/UserManagementTest.kt:1229]: remove this

File: src/main/kotlin/us/frollo/frollosdk/base/Async.kt [hunk 2]
```diff
@@ -0,0 +1,53 @@
+package us.frollo.frollosdk.base
+
+import android.os.Handler
+import android.os.Looper
+import java.util.concurrent.ExecutorService
+import java.util.concurrent.Executors
+
```
Comment (deepakmn86) [inline src/main/kotlin/us/frollo/frollosdk/base/Async.kt:7]: Put a comment with the link to the anko source code from where we picked this code. For future reference.

File: src/main/kotlin/us/frollo/frollosdk/base/Async.kt [hunk 3]
```diff
@@ -0,0 +1,53 @@
+package us.frollo.frollosdk.base
+
```
Comment (deepakmn86) [inline src/main/kotlin/us/frollo/frollosdk/base/Async.kt:2]: Add Copyright header.

File: src/androidTest/kotlin/us/frollo/frollosdk/user/UserManagementTest.kt [hunk 4]
```diff
@@ -1225,6 +1226,7 @@ class UserManagementTest : BaseAndroidTest() {
         tearDown()
     }
```
Comment (deepakmn86) [inline src/androidTest/kotlin/us/frollo/frollosdk/user/UserManagementTest.kt:1228]: - Can we also replace the line `implementation fileTree(dir: 'sdk_libs', include: ['*.aar','*.jar'], exclude: [])` in build.gradle with `implementation fileTree(include: ['*.jar'], dir: 'libs')` ?
- Also remove the todo comment in build.gradle `//todo need to fix test framework with libs in aar/jar`

**Review by deepakmn86 — CHANGES_REQUESTED** [review]

File: src/androidTest/kotlin/us/frollo/frollosdk/user/UserManagementTest.kt (hunk 4 above)
Comment (kfrollo) [reply src/androidTest/kotlin/us/frollo/frollosdk/user/UserManagementTest.kt:1228]: removed the fileTree line completely as its not needed anymore

**Review by deepakmn86 — APPROVED** [review]

**Pull request merged**